setup.py

```diff
@@ -5,26 +5,21 @@
 import re
 
 
-v = open(os.path.join(os.path.dirname(__file__), 'ibm_db_sa', '__init__.py'))
+v = open(os.path.join(os.path.dirname(__file__), 'pyodbc_sa', '__init__.py'))
 VERSION = re.compile(r".*__version__ = '(.*?)'", re.S).match(v.read()).group(1)
 v.close()
 
-readme = os.path.join(os.path.dirname(__file__), 'README.rst')
-if 'USE_PYODBC' in os.environ and os.environ['USE_PYODBC'] == '1':
-    require = ['sqlalchemy>=0.7.3']
-else:
-    require = ['sqlalchemy>=0.7.3','ibm_db>=2.0.0']
+readme = os.path.join(os.path.dirname(__file__), 'README.md')
+require = ['sqlalchemy>=0.7.3', 'pyodbc']
```

Comment:
the version of sqlalchemy is ancient. I suggest we use ">=1.3" instead. If users need earlier, we can test with earlier.

Reply:
I also think this list can be inlined since there's no conditional now